beacon-chain/powchain/service.go

```diff
@@ -141,22 +143,22 @@ func (w *Web3Service) run(done <-chan struct{}) {
 	}
 }
 
-// LatestBlockNumber is a getter for blockNumber to make it read-only.
+// LatestBlockNumber exposes a getter for blockNumber.
```

Comment:
These comments that state that the method is a getter are not helpful. Please change this to 

// LatestBlockNumber in the PoWChain.